Documentation/nvme-connect-all.txt

```diff
@@ -109,6 +110,12 @@ OPTIONS
 	and dump it to a raw binary file. By default 'nvme connect-all' will
 	dump the output to stdout.
 
+-C <cfg>::
+--config-file=<cfg>::
+	Use the specified JSON configuration file instead of the
+	default /etc/nvme/config.json file or 'none' to not read in
```

Comment:
@hreinecke couple of comments:
1. We need this file to be added as part of nvme-cli installation - so pkg-mgmt can upgrade it correctly
2. We need to have an example on how this file looks like - preferably in the documentation
    The reason is that people will want to populate it without the dump command.

Reply:
Well, the config file is optional, so I deliberately did not add it as part of the installation.
If a user wants to use a config file he can generate one; and as it's user-generated we shouldn't update it.
Doing so will create an unholy mess of trying to parse and _update_ user-generated input.

Reply:
But yeah, I already figured we'd need a json schema for the config file.

Reply:
I have submitted a pull request for a json config schema  in https://github.com/linux-nvme/libnvme/pull/102; please check if that's what you had in mind.

Reply:
I still think that it needs to be installed as an empty file and and be removed with the pkg. Primarily because its a default path nvme-cli/libnvme assume and make use of...
The upgrade should do the standard of .rpmsave and/or .rpmnew

Reply:
So, added the 'config.json' file as a '%ghost' file to the rpm template.